package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "eslint-config-edx",
+  "version": "0.0.1",
+  "description": "ESLint config for edX JavaScript code.",
+  "main": "index.js",
+  "scripts": {
+    "test": "eslint ."
```

Comment:
Sorry that wasn't clear. I was thinking we could provide good representative code samples that would pass the linter. The documentation then could point people at that code to say here's how we expect our code to look. These samples would demonstrate the rules that we have changed from AirBnB. 

I was then wondering about negative tests but that seems less important, and is a little complex to make work.